edgelet/edgelet-docker/src/settings.rs

```diff
@@ -493,25 +492,11 @@ mod tests {
         {
             assert_eq!(
                 content_trust_map.get("contoso1.azurcr.io"),
-                Some(&std::path::PathBuf::from("/path/to/root_ca_contoso1.crt"))
+                Some(&String::from("content-trust-contoso1.azurecr.io"))
```

Comment:
`Some("content-trust-contoso1.azurecr.io")`

Same for the statement below.

If it complains about `Option<&String> cannot be compared with Option<&str>`, then do:

```rust
assert_eq!(
    content_trust.get("contoso1.azurecr.io").as_deref(),
    Some("content-trust-contoso1.azurecr.io")
);
```

Reply:
Oh, actually instead of `.as_deref()` it should be `.map(AsRef::as_ref)`

`.as_deref()` would convert `&Option<String>` to `Option<&str>`, whereas you want to convert `Option<&String>` to `Option<&str>`